mono/metadata/icall.c

```diff
@@ -107,6 +107,8 @@
 #include <sys/utsname.h>
 #endif
 
+G_BEGIN_DECLS // lack of prototypes
```

Comment:
I think that is what I did but will double check. It might not matter any longer though & these can dropped. I.e. once the tree is enough C++, there is no need for C/C++ interop, like between mini & metadata. So let me just try dropping these. I do have a test bed in which all this code actually compiles & links. The changes are not hypothetical, or at least were not in the first place, but might no longer be needed.